src/plugins/intel_cpu/src/nodes/executors/acl/acl_pooling.cpp

```diff
@@ -42,8 +42,9 @@ bool AclPoolingExecutor::isSupported(const TensorInfo& srcTensorInfo,
         DEBUG_LOG("Unknown pooling algorithm: ", static_cast<int>(poolingAttrs.algorithm));
         return false;
     }
-    DimensionRoundingType round = (poolingAttrs.rounding == op::RoundingType::CEIL) ?
-                                   DimensionRoundingType::CEIL : DimensionRoundingType::FLOOR;
+
+    if (dataLayout == arm_compute::DataLayout::NCHW && poolingAttrs.rounding == op::RoundingType::CEIL) return false;
```

Comment:
how does CEIL and NHWC work?

Reply:
@ilya-lavrenov I corrected rounding, for other cases.